src/osemgrep/core/Metrics_.ml

```diff
@@ -24,6 +26,8 @@ module Out = Semgrep_output_v1_t
 (*****************************************************************************)
 
 let _metrics_endpoint = "https://metrics.semgrep.dev"
```

Comment:
In OCaml we prefix a variable with an underscore when it's actually not used. 
It's because by default OCaml will complain if a variable is unused, and so you add a _ prefix when it's unused on purpose. I think now you're actually using metrics_endpoint and so on so you can remove the _ prefix.

Reply:
Right now I'm using `Env.v.metrics_url` (so I could test the metrics payloads locally by passing in a custom endpoint) and copied the url over as the default (and did not use the variable).

I kept this var here as a reference but it's currently unused.